test/functionalTest/cases/1156_mq_as_uri_param_for_metadata/mq_as_uri_param_for_metadata.test

```diff
@@ -0,0 +1,807 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+mq as URI param for metadata in filters
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
```

Comment:
Maybe you have already done but just in case... have you tested this .test with brokerStart adjusted for -noCache? Just to be sure that it also works in that way.

(I know, manually testing the -noCache cases is not the best approach... but the only possible right now)